app/src/main/java/com/example/cocktails/ScrollingActivity.kt

```diff
@@ -135,7 +136,7 @@ class ScrollingActivity : AppCompatActivity() {
         val listCocktailType = object : TypeToken<List<Cocktail>>() {}.type
         val items: ArrayList<Cocktail> =
             Gson().fromJson<ArrayList<Cocktail>>(jsonString, listCocktailType)
-
+        items.addAll((applicationContext as Cocktails).mUserCocktailsList)
```

Comment:
don't worry this line is only for my debug . in the next commit ( 2 min ) I will remove this line